src/components/task/repo/task.repo.js

```diff
@@ -0,0 +1,133 @@
+const _ = require('lodash');
+const { SubmissionModel, HomeworkModel } = require('../db');
+
+const { updateManyResult, deleteManyResult } = require('../../helper/repo.helper');
+
+const { isValid: isValidObjectId } = require('../../../helper/compare').ObjectId;
+const { missingParameters } = require('../../../errors/helper/assertionErrorHelper');
+const { AssertionError } = require('../../../errors');
+
+const validateObjectId = (objectId) => {
```

Comment:
this is an object with multiple objectIdProperties, just as reminder : {...objectIds}